scripts/windows/build-dotnet-trace.bat

```diff
@@ -0,0 +1,62 @@
+IF NOT EXIST .\dep\dotnet-install.ps1 (
+    ECHO Downloading dotnet installation script...
+    wget -O .\dep\dotnet-install.ps1 https://dot.net/v1/dotnet-install.ps1
+)
+
+powershell -ExecutionPolicy Bypass -File .\dep\dotnet-install.ps1 -Runtime dotnet -Version 6.0.0 -InstallDir .\dep\dotnet
```

Comment:
```suggestion
SET DOTNET_VERSION=6.0.405
powershell -ExecutionPolicy Bypass -File .\dep\dotnet-install.ps1 dotnet -Version %DOTNET_VERSION% -InstallDir .\dep\dotnet
```
From my observation:
- Dotnet tools won't install if dotnet SDK is not available. Installing with flag `-Runtime` triggers installation of runtime only.
- Specific version of SDK must be given as well; 6.0.0 offers runtime only, no SDK with that version could be downloaded, at least by the tool